install_config/router/default_haproxy_router.adoc

```diff
@@ -455,6 +456,26 @@ a router can filter out the namespaces and/or routes that it should admit.
 This enables you to partition routes amongst multiple router deployments
 effectively distributing the set of routes.
 
+*Router Sharding and DNS*
+
+If you are using wildcard DNS, the administrator is responsible for making sure a separate DNS entries 
+is created for each router.
+
+For example: 
+
+* If Router A lives on host 192.168.0.5 and has routes with *.foo.com 
+* And Router B lives on host 192.168.1.9 and has routes with *.bar.com 
+
+You will need DNS entries: 
+
+* *.foo.com A IN 192.168.0.5 
+* *.bar.com A IN 192.168.1.9
+
```

Comment:
The above is accurate and useful. Admins also use CNAME records to avoid having to specify the IP address. This gives them more flexibility to change router hosts with a change of the A record. All the CNAME records point to the name on the A record.